src/Console/Locale/CopyLocaleCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Console\Locale;
+
+use Backend\Core\Language\Locale;
+use Common\Locale as CommonLocale;
+use Common\ModulesSettings;
+use ForkCMS\Bundle\CoreBundle\Manager\CopyModulesToOtherLocaleManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * This command will copy all the content from all the modules
+ */
+class CopyLocaleCommand extends Command
+{
+    /**
+     * @var CopyModulesToOtherLocaleManager
+     */
+    private $copyModulesToOtherLocaleManager;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $formatter;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    /**
+     * @var ModulesSettings
+     */
+    private $settings;
+
+    public function __construct(
+        ModulesSettings $settings,
+        CopyModulesToOtherLocaleManager $copyModulesToOtherLocaleManager
+    ) {
+        parent::__construct();
+
+        $this->settings = $settings;
+        $this->copyModulesToOtherLocaleManager = $copyModulesToOtherLocaleManager;
+    }
+
+    protected function configure(): void
+    {
+        $this->setName('forkcms:locale:copy')
+            ->setDescription('Copy the content across modules from one locale to another.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $this->formatter = new SymfonyStyle($input, $output);
+
+        $fromLocale = $this->askFromLocale();
+        $toLocale = $this->askToLocale($fromLocale);
+
+        $this->copyModules($fromLocale, $toLocale);
+    }
+
+    private function askFromLocale(): CommonLocale
+    {
+        return Locale::fromString($this->formatter->choice(
+            'What locale would you like to copy?',
+            $this->getActiveLocale()
+        ));
+    }
+
+    private function askToLocale(CommonLocale $fromLocale): CommonLocale
+    {
+        return Locale::fromString($this->formatter->choice(
+            'To which locale would you like to copy it?',
+            $this->getToLocale($fromLocale)
+        ));
+    }
+
+    private function copyModules(CommonLocale $fromLocale, CommonLocale $toLocale)
+    {
+        $this->copyModulesToOtherLocaleManager->copy($fromLocale, $toLocale);
+
+        $this->io->success('All modules are copied from "' . $fromLocale . '" to "' . $toLocale . '".');
```

Comment:
Do all modules support this copy? I don't think that's the case so I would change the returned message to "All supported Modules ..."

Reply:
Fixed